### PR TITLE
fix(blocks): Select without border and disable styles

### PR DESCRIPTION
**Pull request by madhurisandbhor**
### What does it do?

Blocks `Select` should match below styles as per [design](https://www.figma.com/file/MWPKHNT98LLWHGRp3MyWqu/%5BWIP%5D-Content-Studio---Rich-Text-(Blocks)?type=design&node-id=0-1&mode=design&t=XXcnLGhWsTHHJltn-0)
- shouldn’t have a border
- should inherit its cursor when disabled 
- shouldn’t have a background color when disabled

### CODE REVIEW

**Review by simotae14 — COMMENTED** [review]
It works for me but I prefer to see other opinions

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js [hunk 1]
```diff
@@ -42,6 +50,31 @@ const FlexButton = styled(Flex).attrs({ as: 'button' })`
   }
 `;
 
+const SelectWrapper = styled(Box)`
+  // Styling changes to SingleSelect component doesn't work, so adding wrapper to target SingleSelect
```
Comment (remidej) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:54]: just grammar but changes is plural so
```suggestion
  // Styling changes to SingleSelect component don't work, so adding wrapper to target SingleSelect
```

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js [hunk 2]
```diff
@@ -42,6 +50,31 @@ const FlexButton = styled(Flex).attrs({ as: 'button' })`
...
+  & > * div[role='combobox'] {
```
Comment (remidej) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:55]: Is the `> *` necessary? would this work instead?

```
& div[role='combobox'] {
```

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js [hunk 3]
```diff
@@ -42,6 +50,31 @@ const FlexButton = styled(Flex).attrs({ as: 'button' })`
...
+    border: none;
+    cursor: pointer;
+
+    &[aria-disabled='false'] {
+      &:hover {
```
Comment (remidej) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:60]: I think you can combine them:

```
&[aria-disabled='false']:hover {
```

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js (hunk 2 above)
Comment (madhurisandbhor) [reply packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js:55]: Yes, you're right!

**Review by simotae14 — APPROVED** [review]
Well done, thanks @madhurisandbhor

**Pull request merged**